fan-xiaomi.js

```diff
@@ -119,10 +189,10 @@ class FanXiaomi extends HTMLElement {
                     let newAngle
                     if (u.innerHTML == 'On') {
                         hass.callService('fan', 'xiaomi_miio_set_child_lock_off')
-                        u.innerHTML = 'Off'
+                        //u.innerHTML = 'Off'
```

Comment:
Is this unintentional? Similarly below in the toggling-on scenario

Reply:
This is intentional. When we set usuing innerHTML to some value, there is still a read loop which reads from the entity, and overrides it for a split second. So my problem was that changing any value, I saw "On" -> *click* -> "Off" (set by innerHTML line 122) -> "On" (set by some readback loop from entity, because entity state is not yet changed) -> "Off" (readback loop from entity, after entity has changed it's state).

So my idea was to remove this forces setting of the value, and let us rely on things we read from entity. Now it's "On" -> *click* -> "Off" (when state has been set on entity, and entity reported it).

Reply:
I see, yes I agree I've always had this issue with the inconsistent states but haven't figured out how to fix that. Thanks